Configs/.zshrc

```diff
@@ -5,9 +5,16 @@ ZSH=/usr/share/oh-my-zsh/
 source /usr/share/zsh-theme-powerlevel10k/powerlevel10k.zsh-theme
 
 # List of plugins used
-plugins=()
+plugins=(git sudo zsh-256color zsh-autosuggestions zsh-syntax-highlighting)
 source $ZSH/oh-my-zsh.sh
 
+# Dark theme doesn't work great on linux terminal
+if [[ $TERM == 'linux' ]]; then
+    [[ ! -f ~/.p10k.zsh.light ]] || source ~/.p10k.zsh.light
+else
+    [[ ! -f ~/.p10k.zsh.night ]] || source ~/.p10k.zsh.night
+fi
+
```

Comment:
since zsh can be configured during install its better so give this as an option instead of forcing to use dark mode. so i'll look into.